ui/src/app/pages/apps-routes/app-installed-show/app-installed-show.page.ts

```diff
@@ -64,8 +62,12 @@ export class AppInstalledShowPage extends Cleanup {
     this.cleanup(
       markAsLoadingDuring$(this.$loading$, this.preload.appFull(this.appId))
         .pipe(
-          tap(app => this.app = app),
-          concatMap(() => this.syncWhenDependencyInstalls()), //must be final in stack
+          tap(app => {
+            this.app = app
+            const appP = peekProperties(this.app)
+            this.hideLAN = !appP.lanAddress || (appP.id === 'mastodon' && appP.versionInstalled === '3.3.0') // @TODO delete this hack in 0.3.0
```

Comment:
is this the only version, or are subsequent emver revisions also affected?

Reply:
Subsequent versions should have the proper `lanUi` boolean, so only needed for 0.3.0.